docs/retrievalclient.mmd

```diff
@@ -63,6 +63,7 @@ stateDiagram-v2
 	4 --> 8 : ClientEventPaymentChannelErrored
 	5 --> 8 : ClientEventPaymentChannelErrored
 	6 --> 8 : ClientEventPaymentChannelErrored
+	6 --> 13 : <invalid Value>
```

Comment:
Not sure how these `.mmd` files are generated, but this seems wrong, or?

Reply:
Ah good catch, I forgot to add the stringified versoin